app/models/course.rb

```diff
@@ -44,7 +44,7 @@ class Course < ActiveRecord::Base
   # @!method with_owners
   #   Includes all course_users with the role of owner.
   scope :with_owners, (lambda do
-    course_users = CourseUser.owner.where(course: pluck(:id))
+    course_users = CourseUser.owner.with_approved_state.where(course: pluck(:id)).includes(:user)
```

Comment:
OK, I find there are some places where we don't want to eager load  user, like in `Course::UsersController`: https://github.com/Coursemology/coursemology2/blob/master/app/controllers/course/users_controller.rb#L53

Do we still want to add a default_scope when some places don't need ?  And as said, most of the places actually need it.